cortex/skill.py

```diff
@@ -39,22 +37,28 @@ class SkillClient(_Client):
         'undeploy': 'projects/{projectId}/skills/{skillName}/undeploy',
     }
```

Comment:
can we clean up the URIs constant so the string vars in `{}` use either camel case OR snake case but not both